packages/plugin-commands-patching/package.json

```diff
@@ -16,7 +16,7 @@
     "registry-mock": "registry-mock",
     "test:jest": "jest",
     "test:e2e": "registry-mock prepare && run-p -r registry-mock test:jest",
-    "_test": "cross-env PNPM_REGISTRY_MOCK_PORT=7777 pnpm run test:e2e",
+    "_test": "cross-env PNPM_REGISTRY_MOCK_PORT=7776 pnpm run test:e2e",
```

Comment:
This also keeps changing when I am trying to commit or push to my branch not sure why this happens

Reply:
it is a bug in meta-updater.

Reply:
What is the culture on your computer? Maybe it happens because this sort works differently on different cultures: https://github.com/pnpm/pnpm/blob/e19356c32fb87e1555d842cd0fdec7382ce74a91/packages/find-workspace-packages/src/index.ts#L40

Reply:
I would expect it to be `en_GB.UTF-8` (process.env.LANG's value in Node)